[2025-03-29T14:55:43.982+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-29T14:55:44.104+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fist_dag_v5.first_task scheduled__2024-12-19T00:00:00+00:00 [queued]>
[2025-03-29T14:55:44.164+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fist_dag_v5.first_task scheduled__2024-12-19T00:00:00+00:00 [queued]>
[2025-03-29T14:55:44.168+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-03-29T14:55:44.308+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2024-12-19 00:00:00+00:00
[2025-03-29T14:55:44.381+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=186) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-29T14:55:44.429+0000] {standard_task_runner.py:72} INFO - Started process 201 to run task
[2025-03-29T14:55:44.376+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fist_dag_v5', 'first_task', 'scheduled__2024-12-19T00:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpf80zehjo']
[2025-03-29T14:55:44.446+0000] {standard_task_runner.py:105} INFO - Job 140: Subtask first_task
[2025-03-29T14:55:44.831+0000] {task_command.py:467} INFO - Running <TaskInstance: fist_dag_v5.first_task scheduled__2024-12-19T00:00:00+00:00 [running]> on host ba06481f48a9
[2025-03-29T14:55:45.585+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='fist_dag_v5' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-19T00:00:00+00:00'
[2025-03-29T14:55:45.596+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-29T14:55:45.678+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-29T14:55:45.690+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2025-03-29T14:55:45.821+0000] {subprocess.py:99} INFO - Output:
[2025-03-29T14:55:45.837+0000] {subprocess.py:106} INFO - hello world, this is the first task!
[2025-03-29T14:55:45.852+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-29T14:55:46.100+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-29T14:55:46.116+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fist_dag_v5, task_id=first_task, run_id=scheduled__2024-12-19T00:00:00+00:00, execution_date=20241219T000000, start_date=20250329T145544, end_date=20250329T145546
[2025-03-29T14:55:46.899+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-29T14:55:47.004+0000] {taskinstance.py:3895} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-03-29T14:55:47.043+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
